### PR TITLE
Address NPM audit alert 1762 (Uplift to Beta)

### PR DESCRIPTION
**Pull request by diracdeltas**
Fix https://github.com/brave/brave-browser/issues/14423

Uplift of https://github.com/brave/brave-core/pull/9267

### CODE REVIEW

**Review by mihaiplesa — APPROVED** [review]

**Review by kjozwiak — APPROVED** [review]
Uplift into `1.27.x` approved after deliberating with @brave/uplift-approvers.

**Pull request merged**
